server/Cargo.toml

```diff
@@ -50,6 +50,8 @@ assets = [
   ["target/release/innernet-server", "usr/bin/", "755"],
   ["innernet-server@.service", "usr/lib/systemd/system/", "644"],
   ["../doc/innernet-server.8.gz", "usr/share/man/man8/", "644"],
+  ["server.bash", "etc/bash_completion.d/innernet", "644"],
+  ["server.fish", "usr/local/share/fish/vendor_completions.d/innernet-server.fish", "644"],
```

Comment:
Similarly, do we want to copy all the various completions in OS packages, or do we only want to choose a popular subset (eg. bash/fish/zsh)?

I think it makes sense that the completions that live in `doc/` and the completions installed in OS packages should be matching.

Reply:
I only did Bash/Fish since they have standard locations where the OS looks for global completions. The other shells don't have a directory like that. So we could place the scripts for the shells without a global directory somewhere like `/usr/share/innernet/completions/` at which point it's up to the user to do something with those scripts if they wish. It'd just need to be documented somewhere that they're available. I just didn't want to make that assumption without approval first.

Reply:
It looks like zsh can be dropped in `/usr/share/zsh/site-functions/_innernet` (or `_innernet-server`). I think sticking to the holy trinity of zsh, fish, and bash in packages makes sense to me! Also in that referenced line 53, should that be `innernet-server` and not `innernet`?

Reply:
(i assumed this just looking at how `ripgrep` does its packages in arch: https://github.com/archlinux/svntogit-community/blob/packages/ripgrep/trunk/PKGBUILD#L36)

Reply:
Sounds good to me, I'll make that change!